src/main/app/forms/models/cookie.ts

```diff
@@ -0,0 +1,50 @@
+import { RepresentativeDetails } from 'forms/models/representativeDetails'
+
+export class Cookie {
+  representativeDetails: RepresentativeDetails[]
+
+  public static getCookie (legalRepresentativeDetails: RepresentativeDetails[], id: string): RepresentativeDetails {
+    if (legalRepresentativeDetails === undefined) {
+      return new RepresentativeDetails()
+    } else {
+      const representativeDetail: RepresentativeDetails = legalRepresentativeDetails.find(function (representativeDetail: RepresentativeDetails) {
+        return representativeDetail.id === id
+      })
+
+      if (representativeDetail === undefined) {
+        return new RepresentativeDetails()
+      } else {
+        return representativeDetail
+      }
+    }
+  }
+
+  public static saveCookie (legalRepresentativeDetails: RepresentativeDetails[], id: string, legalRepDetails: RepresentativeDetails): RepresentativeDetails[] {
+    let representativeDetails: RepresentativeDetails[] = legalRepresentativeDetails === undefined ? [] : legalRepresentativeDetails
+    representativeDetails = representativeDetails.filter(function (representativeDetail: RepresentativeDetails) {
+      return representativeDetail.id !== id
+    })
+
+    if (legalRepDetails !== undefined) {
+      legalRepDetails.id = id
+      representativeDetails.push(legalRepDetails)
+    }
+
+    return representativeDetails
+  }
+
+  deserialize (input: any): Cookie {
+    if (input) {
+      if (input.files) {
```

Comment:
can we please use single if statement?
```
if (input &&  input.files)
```

Reply:
done. Don't know how I missed something so obvious